### PR TITLE
disable systemd cgroup for the containerd main env

### PR DESCRIPTION
**Pull request by tzneal**

### CODE REVIEW

**Comment by tzneal** [comment]
Partial revert of https://github.com/kubernetes/test-infra/commit/e69360aac36656ad6ef4947399f60dde4d027c0c

**Review by adisky — COMMENTED** [review]
@tzneal can you add PR description? as this is the fix for `containerd-node-e2e-features-1.7`/ `containerd-node-e2e-1.7` failing tests?

Also I think fix could be other way around as well, instead of unsetting systemd here, we can set `--cgroups-driver: systemd` in kubelet flag for failing tests.

File: jobs/e2e_node/containerd/containerd-main/env [hunk 1]
```diff
@@ -6,4 +6,3 @@ CONTAINERD_PKG_PREFIX: 'containerd-cni'
 CONTAINERD_EXTRA_RUNTIME_HANDLER: 'test-handler'
 CONTAINERD_EXTRA_RUNTIME_OPTIONS: |
   BinaryName = "/home/containerd/usr/local/sbin/runc"
-CONTAINERD_SYSTEMD_CGROUP: 'true'
```
[the remark this answers is not in the document]
Comment (tzneal) [reply jobs/e2e_node/containerd/containerd-main/env:9]: It was to resolve the dozens of tests that broke when the default was changed to systemd.

**Pull request closed without merging**